deploy/osd-25526-oao-olm-dance/readme.md

```diff
@@ -0,0 +1,3 @@
+# README
+
+This cronjob is a fix for https://issues.redhat.com/browse/OSD-25526 and ITN-193
```

Comment:
FYI, we can track the OLM dance requirement as per https://issues.redhat.com/browse/OSD-25349